### PR TITLE
Makefile: replace TAP with TAPTOOL

### PR DESCRIPTION
**Pull request by kolyshkin**
Apparently, TAP is used by node-tap, so when it is used to specify a
path to node-tap, this results in a failure:

	$ make TAP=/path/to/node-tap localvalidation
	...
	Error: Environment variable TAP must be set to 0 or 1 only

Fixes: #723

### CODE REVIEW

**Review by dgibson — APPROVED** [review]
Looks good, thanks for making this change.

**Review by tianon — APPROVED** [review]

**Comment by kolyshkin** [comment]
Can we please have a second LGTM on this, @opencontainers/runtime-tools-maintainers ?

**Review by hqhq — APPROVED** [review]

**Pull request merged**
